content/en/docs/helm/helm_delete.md

```diff
@@ -0,0 +1,7 @@
+---
+section: depreciated
+---
+
+## helm delete
+
+This command has been depricated. Please refer instead to [helm uninstall](../helm_uninstall/).
```

Comment:
since `helm delete` was only re-named (and is accepted as an alias), should this be re-worded as "This command has been re-named. Please refer to helm uninstall"?

Reply:
I would prefer the language note it was renamed as that is more clear.